Contributing.md

```diff
@@ -0,0 +1,88 @@
+# Contributing
+
+🎊 🎉 You're Awesome!! 🎉 🎊
+
+Thank you for considering a contribution to our project.
+
+## Installation
+
+If you have never set up Jekyll or Ruby in your local environment, we recommend the Jekyll docs for [installing on your platform.](https://jekyllrb.com/docs/installation/)
+
+We use the [latest stable Ruby version for development](https://github.com/hurricane-response/response-theme/blob/master/.ruby-version)
+
+We also use [Gulp](https://gulpjs.com) for our build tool. Though Jekyll has support for Sass, we have opted to write plain CSS for 2019 browsers and let the tools backfill for compatibility. If you haven't worked with Gulp before, we recommend their [getting started docs](https://gulpjs.com/docs/en/getting-started/quick-start) which includes a link to [Node install steps](https://nodejs.org/en/)
+
+
+Clone the repository
+
+```{bash}
+git clone git@github.com:hurricane-response/response-theme.git
+```
```

Comment:
May want to give the `https` option here as well.

Reply:
I think it requires you enter your password every time you interact is all.